lua/lazyvim/plugins/extras/overseer/core.lua

```diff
@@ -0,0 +1,120 @@
+return {
+  {
+    "catppuccin",
+    optional = true,
+    opts = {
+      integrations = { overseer = true },
+    },
+  },
+  {
+    "folke/which-key.nvim",
+    optional = true,
+    opts = {
+      defaults = {
+        ["<leader>o"] = { name = "+overseer" },
+      },
+    },
+  },
+  {
+    "nvim-lualine/lualine.nvim",
+    optional = true,
+    opts = function(_, opts)
+      local overseer = require("overseer")
+      local lazyvim = require("lazy.core.config").spec.plugins["LazyVim"]
+      local icons = require("lazy.core.plugin").values(lazyvim, "opts", false).icons.overseer
```

Comment:
I get an error on line 24
```
   Error  01:38:42 PM notify.error lazy.nvim Failed to run `config` for lualine.nvim

...ins/LazyVim/lua/lazyvim/plugins/extras/overseer/core.lua:24: attempt to index field 'icons' (a nil value)

# stacktrace:
  - lua/lazyvim/plugins/extras/overseer/core.lua:24 _in_ **values**
```
My workaround was to remove lines 23-24 and replace them with 
```lua
local icons = require("lazyvim.config").icons
```
Do you get the same error?

Reply:
No, but I submited a patch to fix it